Add package ml jobs (#2496)
elastic/stack-docs: docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/datafeed_v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]

|===
// end::security-windows-jobs[]

[discrete]
[[security-integrations-jobs]]
== Security: Elastic Integrations

https://docs.elastic.co/integrations[Elastic Integrations] are a streamlined way to add Elastic assets to your environment, such as data ingestion, transforms, and in this case, machine learning capabilities for Security.

> The following Integrations are for the Security UEBA (user and entity behavioral analytics), which can help detect and alert when there is related suspicious activity in your environment.

Reviewer: ```suggestion
The following Integrations use Machine Learning to analyze patterns of user and entity behavior, and help detect and alert when there is related suspicious activity in your environment.
```

Author: Thank you, updated this (used their convention so I made a separate commit)

* https://docs.elastic.co/integrations/ded[Data Exfiltration Detection]
* https://docs.elastic.co/integrations/dga[Domain Generation Algorithm Detection]
* https://docs.elastic.co/integrations/lmd[Lateral Movement Detection]
* https://docs.elastic.co/integrations/problemchild[Living off the Land Attack Detection]

// tag::security-integrations-jobs[]

> *Domain Generation Algorithm (DGA) Detection (Integration)*

Reviewer 2: Can we remove this "integration" from the heading? Guess we've already introduced it as `Elastic Integrations`.

Reviewer: +1

ML solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.

To download, see the https://docs.elastic.co/integrations/dga[documentation].

|===
|Name |Description |Job |Datafeed

|dga_high_sum_probability
|Detect domain generation algorithm (DGA) activity in your network data.
|https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L23[image:images/link.svg[A link icon]]

> |https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L58[image:images/link.svg[A link icon]]

Reviewer 2: Why these links aren't working?

Author: Hi, thanks for looking, they are working on the [preview](https://stack-docs_2496.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-ml-jobs.html#security-integrations-jobs) linked in the PR body

|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]
|https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/datafeed_v3_windows_rare_user_type10_remote_login.json[image:images/link.svg[A link icon]]

|===
// end::security-windows-jobs[]

[discrete]
[[security-integrations-jobs]]
== Security: Elastic Integrations

> https://docs.elastic.co/integrations[Elastic Integrations] are a streamlined way to add Elastic assets to your environment, such as data ingestion, transforms, and in this case, machine learning capabilities for Security.

Reviewer 3: ```suggestion
https://docs.elastic.co/integrations[Elastic Integrations] are a streamlined way to add Elastic assets to your environment, such as data ingestion, {transforms}, and in this case, {ml} capabilities for Security.
```

The following Integrations use {ml} to analyze patterns of user and entity behavior, and help detect and alert when there is related suspicious activity in your environment.

* https://docs.elastic.co/integrations/ded[Data Exfiltration Detection]
* https://docs.elastic.co/integrations/dga[Domain Generation Algorithm Detection]
* https://docs.elastic.co/integrations/lmd[Lateral Movement Detection]
* https://docs.elastic.co/integrations/problemchild[Living off the Land Attack Detection]

// dga

*Domain Generation Algorithm (DGA) Detection*

> ML solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.

Reviewer 3: ```suggestion
{ml-cap} solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.
```

> ML solution package to detect domain generation algorithm (DGA) activity in your network data. Requires a Platinum subscription.

Reviewer 3: Usually, I try to avoid explicitly mentioning subscription levels as it might be difficult to update every occurrence if the subscription level changes. An alternative solution could be to add a link to the subscription page here – which is the source of truth, like below:
```suggestion
ML solution package to detect domain generation algorithm (DGA) activity in your network data. Refer to the {subscriptions}[subscription page] to learn more about the required subscription.
```

> To download, see the https://docs.elastic.co/integrations/dga[documentation].

Reviewer 3: The preferred term to use is `refer` instead of `see` for [accessibility reasons](https://docs.elastic.dev/content-design/accessibility#terms).
```suggestion
To download, refer to the https://docs.elastic.co/integrations/dga[documentation].
```

|===
|Name |Description |Job |Datafeed

|dga_high_sum_probability
|Detect domain generation algorithm (DGA) activity in your network data.
|https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L23[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/dga/kibana/ml_module/dga-ml.json#L58[image:images/link.svg[A link icon]]

|===

// LotL

*Living off the Land Attack (LotL) Detection*

> ML solution package to detect Living off the Land (LotL) attacks in your environment. Requires a Platinum subscription. (Also known as ProblemChild).

Reviewer 3: As above.

> ML solution package to detect Living off the Land (LotL) attacks in your environment. Requires a Platinum subscription. (Also known as ProblemChild).

Reviewer 3: ```suggestion
{ml-cap} solution package to detect Living off the Land (LotL) attacks in your environment. Requires a Platinum subscription. (Also known as ProblemChild).
```

> To download, see the https://docs.elastic.co/integrations/problemchild[documentation].

Reviewer 3: ```suggestion
To download, refer to the https://docs.elastic.co/integrations/problemchild[documentation].
```

|https://github.com/elastic/integrations/blob/{branch}/packages/problemchild/kibana/ml_module/problemchild-ml.json#262[image:images/link.svg[A link icon]]

|problem_child_high_sum_by_user
|Looks for a set of one or more malicious processes, started by the same user (experimental).
|https://github.com/elastic/integrations/blob/{branch}/packages/problemchild/kibana/ml_module/problemchild-ml.json#L177[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/problemchild/kibana/ml_module/problemchild-ml.json#262[image:images/link.svg[A link icon]]

|problem_child_high_sum_by_parent
|Looks for a set of one or more malicious child processes spawned by the same parent process (experimental).
|https://github.com/elastic/integrations/blob/{branch}/packages/problemchild/kibana/ml_module/problemchild-ml.json#L219[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/problemchild/kibana/ml_module/problemchild-ml.json#262[image:images/link.svg[A link icon]]

|===

// ded

*Data Exfiltration Detection (DED)*

> ML package to detect data exfiltration in your network and file data.

Reviewer 3: ```suggestion
{ml-cap} package to detect data exfiltration in your network and file data.
```

> To download, see the https://docs.elastic.co/integrations/ded[documentation].

Reviewer 3: ```suggestion
To download, refer to the https://docs.elastic.co/integrations/ded[documentation].
```

|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L194[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L304[image:images/link.svg[A link icon]]

|rare-process-writing-to-external-device
|Detects data exfiltration activity by identifying a file write started by a rare process to an external device.
|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L231[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L304[image:images/link.svg[A link icon]]

|high-bytes-written-to-external-device-airdrop
|Detects data exfiltration activity by identifying high bytes written to an external device via Airdrop.
|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L268[image:images/link.svg[A link icon]]
|https://github.com/elastic/integrations/blob/{branch}/packages/ded/kibana/ml_module/ded-ml.json#L304[image:images/link.svg[A link icon]]

|===

// lmd

*Lateral Movement Detection (LMD)*

> ML package to detect lateral movement based on file transfer activity and Windows RDP events.

Reviewer 3: ```suggestion
{ml-cap} package to detect lateral movement based on file transfer activity and Windows RDP events.
```

> To download, see the https://docs.elastic.co/integrations/lmd[documentation].

Reviewer 3: ```suggestion
To download, refer to the https://docs.elastic.co/integrations/lmd[documentation].
```
